README.md

```diff
@@ -55,7 +55,7 @@ make -j8
 sudo make install
 ```
 
-To enable autotype, add `-DWITH_XC_AUTOTYPE=ON` to the `cmake` command. KeePassHTTP support is compiled in by adding `-DWITH_XC_HTTP=ON`. If these options are not specified, KeePassXC will be built without these plugins.
+To enable autotype, add `-DWITH_XC_AUTOTYPE=ON` to the `cmake` command. KeePassHTTP support is compiled in by adding `-DWITH_XC_HTTP=ON`. Yubikey HMAC-SHA1 authentication support is compiled in by adding `-DWITH_XC_YUBIKEY=ON`. If these options are not specified, KeePassXC will be built without these plugins.
```

Comment:
Since we have quite a few options now, this should be a list of all possible options and their defaults (-DWITH_XC_AUTOTYPE defaults to ON now).
A second list with developer options (starting with -DWITH_* instead of -DWITH_XC_*) would be useful, too.